extras/bh1750/bh1750.h

```diff
@@ -0,0 +1,65 @@
+/*
+ * Driver for BH1750 light sensor
+ *
+ * Part of esp-open-rtos
+ * Copyright (C) 2017 Andrej Krutak <dev@andree.sk>
+ * BSD Licensed as described in the file LICENSE
+ *
+ * http://rohmfs.rohm.com/en/products/databook/datasheet/ic/sensor/light/bh1750fvi-e.pdf
+ */
+#ifndef EXTRAS_BH1750_H_
+#define EXTRAS_BH1750_H_
+
+#include <stdint.h>
+#include <stdbool.h>
+#include <time.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define BH1750_ADDR_LO 0x23 // ADDR pin floating/low
+#define BH1750_ADDR_HI 0x5c
+
+// No active state
+#define BH1750_POWER_DOWN 0x00
+
+// Wating for measurment command
+#define BH1750_POWER_ON 0x01
+
+// Reset data register value - not accepted in POWER_DOWN mode
+#define BH1750_RESET 0x07
+
+// Start measurement at 1lx resolution. Measurement time is approx 120ms.
+#define BH1750_CONTINUOUS_HIGH_RES_MODE  0x10
+
+// Start measurement at 0.5lx resolution. Measurement time is approx 120ms.
+#define BH1750_CONTINUOUS_HIGH_RES_MODE_2  0x11
+
+// Start measurement at 4lx resolution. Measurement time is approx 16ms.
+#define BH1750_CONTINUOUS_LOW_RES_MODE  0x13
+
+// Start measurement at 1lx resolution. Measurement time is approx 120ms.
+// Device is automatically set to Power Down after measurement.
+#define BH1750_ONE_TIME_HIGH_RES_MODE  0x20
+
+// Start measurement at 0.5lx resolution. Measurement time is approx 120ms.
+// Device is automatically set to Power Down after measurement.
+#define BH1750_ONE_TIME_HIGH_RES_MODE_2  0x21
+
+// Start measurement at 1lx resolution. Measurement time is approx 120ms.
+// Device is automatically set to Power Down after measurement.
+#define BH1750_ONE_TIME_LOW_RES_MODE  0x23
```

Comment:
Better to declare enums instead of all this macros.
Something like
```C
typedef enum
{
     BH1750_MODE_CONTINUOUS_HIGH_RES = 0x10,
     ...
} bh1750_mode_t;
```